libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -415,19 +457,42 @@ export const appStrings = {
     </UiString>
   ),
 
+  instructionsBmdScanReviewConfirmationPatDevice: () => (
+    <UiString uiStringKey="instructionsBmdScanReviewConfirmationPatDevice">
+      If your selections are correct, use the move input to navigate to the
+      control labelled "my ballot is correct" and then use the select input to
+      confirm your choices and cast your ballot. If there is an error, select
+      the option labelled "my ballot is incorrect" to mark this ballot as
+      incorrect and alert a poll worker.
+    </UiString>
+  ),
+
   instructionsBmdSelectToContinue: () => (
     <UiString uiStringKey="instructionsBmdSelectToContinue">
       Press the select button to continue.
     </UiString>
   ),
 
+  instructionsBmdSelectToContinuePatDevice: () => (
+    <UiString uiStringKey="instructionsBmdSelectToContinuePatDevice">
+      Use the select input to continue.
+    </UiString>
+  ),
+
   instructionsBmdWriteInFormNavigation: () => (
     <UiString uiStringKey="instructionsBmdWriteInFormNavigation">
       Use the up and down buttons to navigate between the letters of a standard
       keyboard. Use the select button to select the current letter.
     </UiString>
   ),
 
+  instructionsBmdWriteInFormNavigationPatDevice: () => (
+    <UiString uiStringKey="instructionsBmdWriteInFormNavigationPatDevice">
+      Use the move input to navigate between the letters of a standard keyboard.
```

Comment:
I'm not actually sure if QWERTY keyboards are standard for PAT users. They very well could be, but it might be more specific to say QWERTY keyboard (if our TTS can handle that).

Reply:
Good point -- the audio-only virtual keyboard experience is currently incomplete/not ideal for both tactile controller voters and PAT device voters and we've talked about changing the scroll order to alphabetical for audio-only users, or implementing some T9-type UX for them. We'll revisit these instructions at that point and can treat these as short-term stand-ins for now